apps/frontend/src/pages/analytics/libs/constants/constants.ts

```diff
@@ -1,3 +1,4 @@
 export { ANALYTICS_DATE_MAX_RANGE } from "./analytics-date-max-range.constant.js";
 export { ANALYTICS_DEFAULT_DATE_RANGE } from "./analytics-default-date-range.constant.js";
 export { ANALYTICS_LOOKBACK_DAYS_COUNT } from "./analytics-lookback-days-count.constant.js";
+export { EMPTY_LENGTH } from "@git-fit/shared";
```

Comment:
this should be re-exported from src/libs/constants not from this module

Reply:
oops, thanks